### PR TITLE
fix: external ServiceNow testability

### PR DESCRIPTION
**Pull request by alespour**
Fixes external testability of ServiceNow handler (needed for Chronograf issue #2620).

###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass

### CODE REVIEW

File: services/servicenow/service.go [hunk 1]
```diff
@@ -73,18 +73,25 @@ func (s *Service) StateChangesOnly() bool {
...
 type testOptions struct {
-	AlertID string      `json:"alert_id"`
-	Source  string      `json:"source"`
-	Level   alert.Level `json:"level"`
-	Message string      `json:"message"`
+	AlertID        string                 `json:"alert_id"`
+	Message        string                 `json:"message"`
+	Level          alert.Level            `json:"level"`
+	Source         string                 `json:"source"`
+	Node           string                 `json:"node"`
```
Comment (sranka) [inline services/servicenow/service.go:80]: These new properties are not used in the Test function yet.

Comment (alespour) [reply services/servicenow/service.go:80]: I hope it's fixed now.

**Review by sranka — APPROVED** [review]

**Review by sranka — CHANGES_REQUESTED** [review]
@alespour please rebase and change PR onto the master branch, nothing should be pointing the v1.6-master anymore per https://github.com/influxdata/kapacitor/pull/2542#issuecomment-832359540

**Comment by alespour** [comment]
@sranka thanks. Done.

**Comment by alespour** [comment]
> @alespour please rebase and change PR onto the master branch, nothing should be pointing the v1.6-master anymore per [#2542 (comment)](https://github.com/influxdata/kapacitor/pull/2542#issuecomment-832359540)

Thank you. It is done.

**Review by sranka — APPROVED** [review]

**Pull request merged**